core-skills/c-sharp-and-dotnet/README.md

```diff
@@ -0,0 +1,78 @@
+# C# and .NET Core
+
+## Who is this skill for?
+
+This skill is intended for developers who have already completed the TDD core skill, and who have not worked with .NET previously (or want to brush up) and are likely to join a delivery team that is working in C#.
+Through this course, we'll be making some assumptions that you're already familiar with core Made Tech processes and at least one programming language (static typed or dynamic).
+
+## Summary
+### Mantis Shrimp
+1. Can set up a working project in .NET using an IDE
+2. Can install and manage dependancies for the project
+3. Can create and run unit tests using a test framework
+4. Understands basic concepts of the C# language and ecosystem
+
+### Goat
+1. Understands some deeper object oriented programming concepts
```

Comment:
this is vague

Reply:
Intentionally so, this is a summary of where you'll be at when you get the Goat sticker. Is that something we need to be clearer on?